### DISCUSSION TITLE
Deploy Website by flutter

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143007] [New to GitHub] [answered]
How can a website written in flutter run on a website with the .github.io extension like a html css website?
[](https://github.com/Mukhtar-Ali-Khan/personal-portfolio-website)

**Comment by datonefeed** [comment]
You just need to add index.html to your config file.

**Reply by ghost** [reply]
ok! thank men

**Comment by ebndev** [comment, marked as answer]
Hi @ThanhDatDev11 & @datonefeed, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
